### ISSUE TITLE
Rollout of centraldashboard

### ISSUE THREAD

**Issue by wg102** [StatCan/aaw-argocd-manifests#32]
Migrate the kubeflow folder in the manifest repo.
Use overlays to use more appropriate sizes for dev vs prod.

**Comment by sylus** [comment]
I was thinking the kubeflow folder would be deprecated as we move to argoflow. Is this an intermediate step?

**Comment by blairdrummond** [comment]
I am actually thinking that we change argoflow to point to the kubeflow folder in this repo. Atm we aren't using anything else in the argoflow repo really (no Istio, prometheus, etc) so we can use argoflow for a mock environment but not actually deploying it to dev or prod yet. The Argoflow Application CR can point to this folder here though, so that the aaw-argocd-manifests live in one place.

Just my thinking atm, open to alternatives

**Comment by sylus** [comment]
Is this a change in direction? I was thinking the move to ArgoFlow was to simplify the whole updating process as we would now be calling individual components?

**Comment by sylus** [comment]
@brendangadd ^

**Comment by sylus** [comment]
I did an initial PoC for how should deploy Kubeflow basically we should follow 1 to 1 how Kubeflow upstream organizes there folders and call the naming exactly.

I then similar to how we did the other releases created a stacks folder which will call the different supported flows (AAW, Argo, and Upstream).

This way we can keep all your argoflow work and proceed methodically component by component, just how we did the other versions. Basically the mantra is always call the upstream resources at a pinned version (ie v1.4.1) using Kustomize and then we sprinkle in our deltas component by component.

I was thinking we can go component by component and create an epic issue with a table with such columns as done by AAW by team and possibly then assessed by CNS. We could then get all this completed by 2 weeks or so.

- https://github.com/StatCan/aaw-kubeflow-manifests/tree/feat-poc-v1.4.1

**Workflow**

Select a component
Compare with our kubeflow-manifests repo and your work on aaw-argoflow-azure
Check for any additional adjustments made in current cluster
Write a P.R. for the component and mark for review by CNS for anything maybe missing
Close off the component and proceed to next one
